docs/documentation/integrations.md

```diff
@@ -28,7 +28,7 @@ Usually, this involves a few steps:
   `static/images/integrations/logos/<name>.svg`, where `<name>` is the
   name of the integration, all in lower case; you can usually find them in the
   product branding or press page. Make sure to optimize the SVG graphic by
-  running `svgo -f path-to-file`.
+  running `node_modules/.bin/svgo -f path-to-file`.
```

Comment:
```suggestion
  running `yarn run svgo -f path-to-file`.
```

This works from any working directory.

Reply:
Thanks @andersk, have just updated it to the suggested command instead. Didn't know that was possible, but tested and it did work for me too. Thanks